CHANGELOG.md

```diff
@@ -32,6 +43,13 @@ The format is almost based on [Keep a Changelog](https://keepachangelog.com/en/1
 - Search: Add folder search (#1215)
 - Improve test coverage (#1263)
 - Allow directly adding a feed without going through the discovery process (#1265)
+- Implemented sharing news items between nextcloud users (#1191)
+- Updated the news items table in DB to include sharer data (#1191)
+- Added route for sharing news items (#1191)
+- Added share data in news items serialization (#1191)
+- Added tests for the news items share feature (#1191)
+- Added sharing articles with nextcloud users (#1217)
+- Added sharing articles on social media (Facebook, Twitter) or mail (#1217)
```

Comment:
This is in 16.x, which is unreleased

Reply:
Hmm I don't know how that ended up being there, the idea was to take the file from the master branch 🤔

Reply:
I know why because it was merged into master like that some time ago ..

Reply:
Do you have a moment to fix it in master?

Reply:
https://github.com/nextcloud/news/pull/1314 yup fix is in, fixed this pr too although the changelog here doesn't matter that much